pkg/api/types.go

```diff
@@ -835,6 +838,9 @@ type PodStatus struct {
 	HostIP string `json:"hostIP,omitempty"`
 	PodIP  string `json:"podIP,omitempty"`
 
+	// date and time at which the object was acknowledged by the Kubelet
```

Comment:
I think you should expand a little on "acknowledged by the Kubelet." At least mention that this is before Kubelet tries to pull the container image.